controllers/submariner/route_agent_resources.go

```diff
@@ -92,6 +92,21 @@ func newRouteAgentDaemonSet(cr *v1alpha1.Submariner, name string) *appsv1.Daemon
 							Path: "/var/run/ovn-ic/ovnnb_db.sock",
 						}}},
 					},
+					InitContainers: []corev1.Container{
+						{
+							Name:            name + "-init",
+							Image:           getImagePath(cr, opnames.RouteAgentImage, names.RouteAgentComponent),
+							ImagePullPolicy: images.GetPullPolicy(cr.Spec.Version, cr.Spec.ImageOverrides[names.RouteAgentComponent]),
+							Command:         []string{"kubectl", "wait", "node/$(NODE_NAME)", "--for", "condition=ready"},
```

Comment:
Instead of using a new kubectl binary, can we not use the same route-agent pod as an initContainer where we set some env' variable (similar to how we are [doing in Uninstall](https://github.com/submariner-io/submariner/blob/b9f63cda37c4810dae9d54fd76cb809727119dcb/pkg/routeagent_driver/main.go#L136)) and when the variable is set, it simply reads the [node status](https://github.com/submariner-io/submariner/blob/devel/pkg/node/node.go) and checks if its Ready?

Reply:
I thought of doing that in a second phase but it does make the fix simpler to ship, so I’ll do that now.